src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -15,7 +15,7 @@
 (react/defc Page
   {:render
    (fn [{:keys [state refs this]}]
-     [:div {:style {:textAlign "center"}}
+     [:div {:style {:textAlign "center" :width "50vw"}}
```

Comment:
The element does show up, but something in the layout is off.  It's hard to tell what, because it doesn't handle the alpha very well anyway.  Basically, trying to print with a modal up is totally screwed up no matter what the width element is.